packages/react-native/ReactCommon/react/renderer/textlayoutmanager/platform/ios/react/renderer/textlayoutmanager/RCTTextLayoutManager.h

```diff
@@ -44,6 +44,11 @@ using RCTTextLayoutFragmentEnumerationBlock =
                                                   (facebook::react::ParagraphAttributes)paragraphAttributes
                                                              size:(CGSize)size;
 
+- (float)getBaselineForAttributedString:(facebook::react::AttributedString)attributedString
+                    paragraphAttributes:
+                        (facebook::react::ParagraphAttributes)paragraphAttributes
+                                   size:(CGSize)size;
+
```

Comment:
This and function implementing is dead now, right?

Reply:
Yes, removed in https://github.com/facebook/react-native/pull/45102/commits/418f1e0944ebc0547044e6f9abf58702cfd7e32b